docs/pages/integration-test/masonry.js

```diff
@@ -64,12 +101,17 @@ export default function TestPage(): Node {
           externalCache={booleanize(externalCache)}
           finiteLength={booleanize(finiteLength)}
           flexible={booleanize(flexible)}
-          initialItems={generateExampleItems({ name: 'InitialPin' })}
+          initialItems={
+            realisticPinHeights
+              ? generateRealisticExampleItems({ name: 'InitialPin', pinHeightsSample })
+              : generateExampleItems({ name: 'InitialPin' })
+          }
           manualFetch={booleanize(manualFetch)}
           MasonryComponent={Masonry}
           measurementStore={measurementStore}
           noScroll={booleanize(noScroll)}
           offsetTop={offsetTop}
+          pinHeightsSample={realisticPinHeights ? pinHeightsSample : undefined}
```

Comment:
Do the heights have to be passed separately?

Reply:
"passed separately", as opposed to…?

Reply:
Like, as part of the pins?

Reply:
We could potentially, sure. When we call [MasonryContainer.loadItems](https://github.com/pinterest/gestalt/pull/2823/files#diff-783f4d96975c1127b6aba8b861cf98da9aa9a1ff4b176e003993ffcaa8f1ebf9L198), for non-realistic pins we generate the next batch of pins right then. Since that logic still exists, I chose to keep the realistic pins as similar as possible. The one exception is that we have to have a static set of pin heights (we can't generate those on both the server and the client, or there would be a mismatch), so we pass the generated pin heights through. That allows us to call `generateRealisticExampleItems` where we would otherwise call `generateExampleItems`.